### ISSUE TITLE
Normal Flash settings for Ruffle

### ISSUE THREAD

**Issue by ITawTaw** [ruffle-rs/ruffle#8414]
In a lot of flash games, especially ones with a speedrunning community behind them, settings like quality or 100% view must be enabled. There are certain tricks that can't be performed because of the slight differences with these features enabled. With the growing TAS community for flash games switching from gnash to ruffle, this feature would allow the most accurate representation of a perfect run.

**Comment by n0samu** [comment]
Even if the settings were available, I don't think the subtle behavior changes can be emulated easily.

**Comment by ITawTaw** [comment]
> Even if the settings were available, I don't think the subtle behavior changes can be emulated easily.

Yes they can, libTAS does a really good job of being accurate already and i'm sure the added features would work fine.

**Comment by n0samu** [comment]
You misunderstood what I said. For example, you (or someone else with the same suggestion as you) said this:
> well for many games, flash calculates hitbox different depending on 100% vs show all.

We could implement the 100% view option, but that doesn't mean hitbox calculation would magically change too, unless we somehow figured out exactly why this occurs in Flash Player, tested the exact behavior and figured out how to implement it. It would be way more work than just implementing the 100% feature.

**Comment by ITawTaw** [comment]
Ok I see, I still think this would be a benefit despite the work but I understand if it is a lot just for the setting.

**Comment by ActionWavele** [comment]
The TAS community expects a lot
I think the TAS community should help develop Ruffle
